features_hq/README.md

```diff
@@ -0,0 +1,60 @@
+Cucumber Features
```

Comment:
I wouldn't copy the readme. I would have 1 line referring to the authoritative source